zebra/zebra_rib.c

```diff
@@ -659,13 +659,14 @@ void zebra_rib_evaluate_rn_nexthops(struct route_node *rn, uint32_t seq)
 				char buf1[PREFIX_STRLEN];
 				char buf2[PREFIX_STRLEN];
 
-				zlog_debug("%u:%s has Nexthop(%s) Type: %s depending on it, evaluating %u:%u",
-					   zvrf->vrf->vrf_id,
-					   srcdest_rnode2str(rn, buf1,
-						      sizeof(buf1)),
-					   prefix2str(p, buf2, sizeof(buf2)),
-					   rnh_type2str(rnh->type),
-					   seq, rnh->seqno);
+				zlog_debug(
+					"%s(%u):%s has Nexthop(%s) Type: %s depending on it, evaluating %u:%u",
+					zvrf_name(zvrf), zvrf->vrf->vrf_id,
```

Comment:
zvrf_id?

Reply:
Could you elaborate on?

Reply:
There's a function called `zvrf_id` look in zebra_vrf.h

Reply:
Oh ok, now I got it.